cypress/integration/features/deals/create-partnered-deal-wizard.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Create Partnered Deal
+  Background:
+    Given I navigate to create partnered deal wizard
+
+  Scenario: Go back to deal type select
+    Given I am presented with Partnered Deal proposal stage
+    When I go to previous step
+    Then I am presented the option to choose a partner
+
+  Scenario: I proceed from proposal stage without filling in required fields
+    When I try to proceed to next step
+    Then I am presented with Partnered Deal proposal stage
+    And I am presented with errors proposal stage required errors
+
+  Scenario: I proceed from proposal stage after filling required fields with too short summary and description
+    When I fill in proposal title correctly
+    And I fill in proposal summary with text that is too short
+    And I fill in proposal description with text that is too short
+    And I try to proceed to next step
+    Then I am presented with Partnered Deal proposal stage
+    And I am presented with too short input errors for proposal summary and description
+
+  Scenario: I proceed from proposal stage after filling required fields correctly
+    When I fill in proposal title correctly
+    And I fill in proposal summary with text that meets requirements
+    And I fill in proposal description with text that meets requirements
+    And I try to proceed to next step
+    Then I am presented with Partnered Deal proposal lead stage
+
+  Scenario: Wizard stepper contains correct stages
+    Then I can see stages correct for Partnered Deal
+
+  Scenario: I can navigate the stages freely using stepper
```

Comment:
This somehow sounds like it should be a dedicated component test of the stepper, as the underlying logic does not really require the wizard

Reply:
Not really, because it's the wizard that is handling the navigation.
```
  public goToStage(wizardManager: any, index: number): void {
    const wizardState = this.getWizardState(wizardManager);
    wizardState.indexOfActive = index;

    const params = {
      ...this.router.currentInstruction.params,
      [STAGE_ROUTE_PARAMETER]: wizardState.stages[index].route,
    };

    this.router.navigateToRoute(
      this.router.currentInstruction.config.name,
      params,
    );
  }
```
If something breaks here (maybe someone decides to add validation here) it will break test.

Reply:
I think we should have:
- e2e test for pStepper component
- e2e test for using a the stepper in all Wizards
- unit tests for `goToStage` method and other methods in wizardService

Reply:
Ah okay, I somehow thought stepper will always change url, but it doesn't have to.

That last suggestion makes sense 👍

Reply:
Cool